py4cast/io/outputs.py

```diff
@@ -28,7 +27,7 @@ class GribSavingSettings:
     """
 
     template_grib: str
-    directory: str
+    output_dir: str
```

Comment:
I would prefer using the `pathlib` librairy to manage paths instead of str, see bellow.